board/safety/safety_honda.h

```diff
@@ -239,6 +239,14 @@ static int honda_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
     }
   }
 
+  // Bosch supplemental control check
+  if (addr == 0xE5) {
+    int e5_state = (GET_BYTE(to_send, 0) << 24U) | (GET_BYTE(to_send, 1) << 16U) | (GET_BYTE(to_send, 2) << 8U) | GET_BYTE(to_send, 3);
```

Comment:
use `GET_BYTES_04` macro instead?

Reply:
Tried that. Doesn't allow the message through.
`242   // Bosch supplemental control check
243   if (addr == 0xE5) {
244     int e5_state = GET_BYTES_04(to_send);
245     if (e5_state != 0x04008010) {
246       tx = 0;
247     }
248   }`

Reply:
It's an endianness issue. Try this:
```
if (addr == 0xE5) {
  if (GET_BYTES_04(to_send) != 0x10800004) {
    tx = 0;
  }
}
```

Reply:
Came to the same conclusion. This works.